cmd/provisioner-localpv/app/provisioner.go

```diff
@@ -113,13 +117,18 @@ func (p *Provisioner) Provision(opts pvController.VolumeOptions) (*v1.Persistent
 
 	//TODO: Determine if hostpath or device based Local PV should be created
 	stgType := pvCASConfig.GetStorageType()
+	size := resource.Quantity{}
+	reqMap := pvc.Spec.Resources.Requests
+	if reqMap != nil {
+		size = pvc.Spec.Resources.Requests["storage"]
+	}
+	sendEventOrIgnore(name, size.String(), stgType, analytics.VolumeProvision)
```

Comment:
make the stgType => `local-<stgType>` to keep it consistent with pvType used in the deprovision events.

Reply:
It's already done in the sendEventOrIgnore function, there's an if check for `VolumeProvision`, I.e. `local` is only prefixed when a volume is being provisioned